sites/main-page-title/main-page-title-align-right-en.md

```diff
@@ -0,0 +1,19 @@
+---
+altLangPage: "main-page-title-align-right-fr.html"
+dateModified: "2024-09-26"
+language: "en"
+pageclass: "wb-prettify all-pre"
+layout: without-h1
+title: "Main page title aligned on the right"
+index_json: index.json-ld
+breadcrumbs: [
+		{ "title": "Titre principal - Documentation", "link": "sites/main-page-title/main-page-title-en.html" }
```

Comment:
Please adjust the title of the breadcrumb link to be in English.

Reply:
Done. Thanks